doc/interface/binding/multi_component_bi_langmuir_ldf.rst

```diff
@@ -0,0 +1,44 @@
+.. _multi_component_bi_langmuir_ldf_config:
+
+Multi Component Bi-Langmuir LDF
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+**Group /input/model/unit_XXX/adsorption – ADSORPTION_MODEL = MULTI_COMPONENT_BILANGMUIR_LDF**
+
+
+``IS_KINETIC``
+   Selects kinetic or quasi-stationary adsorption mode: 1 = kinetic, 0 =
+   quasi-stationary. If a single value is given, the mode is set for all
+   bound states. Otherwise, the adsorption mode is set for each bound
+   state separately.
+
+===================  =========================  =========================================
+**Type:** int        **Range:** {0,1}  		**Length:** 1/NTOTALBND
+===================  =========================  =========================================
+
+``MCBLDF_KEQ``
+   Adsorption rate constants in state-major ordering
+
+**Unit:** :math:`m_{MP}^3~mol^{-1}`
+
+===================  =========================  =========================================
+**Type:** double     **Range:** :math:`\ge 0`   **Length:** NCOMP
```

Comment:
Shouldn't the length of `KEQ` be the number of binding sites times the number of components? 
I've also checked the documentation of the regular [Bi-Langmuir model](https://cadet.github.io/master/interface/binding/multi_component_bi_langmuir.html#multi-component-bi-langmuir-config) and I also see the same issue here. Even when reading the parameters. 
If this is indeed not checked correctly, this is another candidate for our ongoing efforts in [improving parameter checks](https://github.com/modsim/CADET/issues/28).